src/panels/NightSkyPanel/tabs/TimeTab.tsx

```diff
@@ -0,0 +1,188 @@
+      <Divider />
+      <Title order={3} my={'xs'}>
+        Jumps
+      </Title>
+      <Stack>
+        <Group gap={'lg'}>
+          <Group gap={'xs'}>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.siderealDayIncrease')}
+            >
+              + Sidereal day
+            </Button>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.siderealDayDecrease')}
+            >
+              - Sidereal day
+            </Button>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.siderealWeekIncrease')}
+            >
+              + Sidereal week
+            </Button>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.siderealWeekDecrease')}
+            >
+              - Sidereal week
+            </Button>
+          </Group>
+          <Group gap={'xs'}>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.SolarDayIncrease')}
+            >
+              + Solar day
+            </Button>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.SolarDayDecrease')}
+            >
+              - Solar day
+            </Button>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.SolarWeekIncrease')}
+            >
+              + Solar week
+            </Button>
+            <Button
+              onClick={() => luaApi.action.triggerAction('os.time.SolarWeekDecrease')}
+            >
+              - solar week
+            </Button>
+          </Group>
+        </Group>
+        <Divider />
+        <Title order={3}>Diurnal Motion</Title>
+        <Group gap={'xs'}>
+          <Button
+            onClick={() => {
+              luaApi.time.interpolatePause(false);
+              luaApi.time.interpolateDeltaTime(1);
+            }}
+          >
+            Realtime
+          </Button>
+          <Button
+            onClick={() => {
+              luaApi.time.interpolatePause(false);
+              luaApi.time.interpolateDeltaTime(60);
+            }}
+          >
+            60x (1 min/sec)
+          </Button>
+          <Button
+            onClick={() => {
+              luaApi.time.interpolatePause(false);
+              luaApi.time.interpolateDeltaTime(120);
+            }}
+          >
+            120x (2 min/sec)
+          </Button>
+          <Button
+            onClick={() => {
+              luaApi.time.interpolatePause(false);
+              luaApi.time.interpolateDeltaTime(300);
+            }}
+          >
+            300x (5 min/sec)
+          </Button>
+
+          <Button onClick={() => luaApi.time.togglePause()}>Play / Pause</Button>
+        </Group>
+      </Stack>
+      <Divider my={'xs'} />
+      <Alert title={'Note'}>
+        <Text>
+          Only some controls are found here. For more control over time, use the Time
+          Panel.
```

Comment:
Do we want to add the calendar icon like we did in the other place for GeoLocation?

Reply:
I Could add the Icon to the alert itself instead?  Din't quite like having it in the text, since that will be annoying once we localize the string